config/jobs/kubernetes/sig-cluster-lifecycle/kubespray.yaml

```diff
@@ -0,0 +1,28 @@
+# kubeadm release and master tests
+periodics:
+  kubernetes-incubator/kubespray:
+  - name: ci-kubernetes-e2e-kubeadm-gce-master
+    interval: 2h
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: quay.io/kubespray/kubespray:v2.7
+        args:
+        - --repo=k8s.io/kubernetes=master
+        - --repo=github.com/kubernetes-incubator/kubespray=master
+        - --timeout=320
+        - --upload=gs://kubernetes-jenkins/logs
+        - --scenario=kubernetes_e2e
+        - --
+        - --cluster=
+        - --deployment=kubespray
+        - --kubeadm=ci
```

Comment:
this arg seems to be required by this tool, which is also part of the chain:
https://github.com/kubernetes/test-infra/blob/f38b4fae1283590c79adda7eebbd156d4fcabb2c/scenarios/kubernetes_e2e_test.py

Reply:
here also notice these two flags:
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm.yaml#L100-L101

we probably want to add the exact same args here in the kubespray job.